sdks/python/tox.ini

```diff
@@ -53,6 +53,16 @@ commands =
   python setup.py nosetests
   {toxinidir}/scripts/run_tox_cleanup.sh
 
+[testenv:py3]
+setenv =
+    BEAM_EXPERIMENTAL_PY3=1
+commands =
+  python --version
```

Comment:
How do we tell Jenkins which version of Python interpreter will be used here? Is it inferred from the name of the test suite (`testenv:py3`)?

Reply:
Yes, locally it works like that (using the default PY3 version on the machine).
It seems like the PY3 tests didn't run on Jenkins: I don't see a `py3 create` line in the log.